src/routes/+layout.svelte

```diff
@@ -1,18 +1,31 @@
 <script lang="ts">
+  import "../app.scss";
+
   import type { PageMetadataMap, PageMetadataMapItem } from "$lib/loadPageMetadataMap";
 
   import { setContext } from "svelte";
   import { afterNavigate } from "$app/navigation";
   import { navigating, page } from "$app/stores";
+  import { browser } from "$app/environment";
 
-  import "../app.scss";
+  import { PUBLIC_VERCEL_SPEED_INSIGHTS_ANALYTICS_ID as analyticsID } from "$env/static/public";
+
+  import { webVitals } from "$lib/vitals";
   import Header from "$lib/components/Header";
   import Footer from "$lib/components/Footer";
   import { intersectionObserverSupport, lazyImageLoadingSupport } from "$lib/constants/support";
   import { RootIntersectionObserver } from "$lib/components/IntersectionObserver";
   import { BlurhashRenderer } from "$lib/components/Image";
   import { key as pageMetadataMapKey } from "$lib/context/pageMetadataMap";
 
+  $: if (browser && analyticsID) {
+    webVitals({
+      path: $page.url.pathname,
+      params: $page.params,
+      analyticsID,
+    });
+  }
```

Comment:
We'll want to check that this isn't firing more than once after we deploy.

Reply:
This will fire whenever `$page` changes, so client-side routing will also be included. This seems to be intended because it matches the behavior of Vercel's example, but we can always disable it if it's not working the way we want.